dali/operators/image/color/brightness_contrast.h

```diff
@@ -91,9 +91,23 @@ class BrightnessContrastOp : public Operator<Backend> {
 
   void AcquireArguments(const workspace_t<Backend> &ws) {
     auto curr_batch_size = ws.GetInputBatchSize(0);
-    this->GetPerSampleArgument(brightness_, "brightness", ws, curr_batch_size);
-    this->GetPerSampleArgument(brightness_shift_, "brightness_shift", ws, curr_batch_size);
-    this->GetPerSampleArgument(contrast_, "contrast", ws, curr_batch_size);
+    if (this->spec_.ArgumentDefined("brightness")) {
+      this->GetPerSampleArgument(brightness_, "brightness", ws, curr_batch_size);
+    } else {
+      brightness_ = std::vector<float>(curr_batch_size, 1.f);
+    }
+
+    if (this->spec_.ArgumentDefined("brightness_shift")) {
+      this->GetPerSampleArgument(brightness_shift_, "brightness_shift", ws, curr_batch_size);
+    } else {
+      brightness_shift_ = std::vector<float>(curr_batch_size, 0);
```

Comment:
Shouldn't it just get a default value from schema? Why we need this here?

Reply:
Because getting which is not present in the schema (e.g. it's `Contrast` op and we fetch for `brightness`)  results in exception.

Reply:
Can we have one constant that defines this default value here and in the schema?

Reply:
done

Reply:
I tend to disagree. The code with 0 and 1 is easier to understand. BTW - wouldn't it just work without the check `if (this->spec_.ArgumentDefined("brightness_shift")) {`?

Reply:
I would agree that it doesn't make much sense to name those constants, but in this case it connects the default values in the schema with the ones here, so I think it  useful.